packages/get-github-info/src/index.ts

```diff
@@ -2,6 +2,8 @@
 import fetch from "node-fetch";
 import DataLoader from "dataloader";
 
+const validRepoNameRegex = /^[A-Za-z0-9_.-]+\/[A-Za-z0-9_.-]+$/m;
```

Comment:
This should be the equivalent
```suggestion
const validRepoNameRegex = /^[\w.-]+\/[\w.-]+$/;
```

Reply:
why don't we support @ ? like '@ant-design/xxx'